### PR TITLE
Fix #132: Added copy to clipboard feature in command

### LINKED ISSUE #132: Add copy to clipboard feature in command
### Deliverables
- [x] Add support for copying to clipboard for all the commands listed on the docs page. 
- [x] Add an icon for copying the text.
- [x] Show a successful message ` Command copied to clipboard`.

@RishabhJain2018

### PR DESCRIPTION
**Pull request by vkartik97**
Fixes: #132
screenshot from 2019-02-08 14-00-34
screenshot from 2019-02-08 14-00-38

### CODE REVIEW

**Comment by vkartik97** [comment]
@RishabhJain2018 Please review!
Also, should I remove `Command Copied to Clipboard!` alert?

File: docs/assets/active.js [hunk 1]
```diff
@@ -14,3 +14,17 @@ function scroll_to_anchor(anchor_id){
...
+
+function CopyToClipboard(elementId) {
+  var aux = document.createElement("input");
+  aux.setAttribute("value", document.getElementById(elementId).innerHTML);
+  document.body.appendChild(aux);
+  aux.select();
+  document.execCommand("copy");
+  document.body.removeChild(aux);
+//   alert("Command Copied to Clipboard!")
+}
```
Comment (RishabhJain2018) [inline docs/assets/active.js]: Please add a new line here.

File: docs/index.html [hunk 2]
```diff
@@ -241,16 +286,25 @@ <h4 class="code-container-style"><b>View the leaderboard of Challenge 1 with Cha
...
+                    <button type="button" class="btn copybtn" onclick="CopyToClipboard('viewLeaderBoard')" data-toggle="tooltip" data-placement="top" title="Copy to Clipboard"><i class="fas fa-copy"></i>
+                    </button>
+                </div>
             </div>
             <div class="row">
                 <div class="col-md-12" id="code-label">
                     <h4 class="code-container-style"><b>View all the submissions to Phase 4 of the challenge 1</b></h4>
                 </div>
             </div>
+            <!-- --------------------------- -->
```
Comment (RishabhJain2018) [inline docs/index.html]: Please remove this comment

Comment (vkartik97) [reply docs/index.html]: Thanks you for pointing that out!

**Comment by vkartik97** [comment]
@RishabhJain2018 
Updated Screenshots:
screenshot from 2019-02-08 14-33-06
screenshot from 2019-02-08 14-33-09

**Comment by RishabhJain2018** [comment]
@vkartik97 Can you please resolve the conflicts?

File: docs/assets/active.js [hunk 3]
```diff
@@ -14,3 +14,17 @@ function scroll_to_anchor(anchor_id){
...
+
+function CopyToClipboard(elementId) {
+  var aux = document.createElement("input");
+  aux.setAttribute("value", document.getElementById(elementId).innerHTML);
+  document.body.appendChild(aux);
+  aux.select();
+  document.execCommand("copy");
+  document.body.removeChild(aux);
+  $.notify("Command Copied to Clipboard",  "success");
+}
```
Comment (RishabhJain2018) [inline docs/assets/active.js]: Please add a new line here

**Comment by RishabhJain2018** [comment]
@vkartik97 Please resolve the conflicts, so that we can merge this.

**Comment by vkartik97** [comment]
@RishabhJain2018 Please review

**Comment by RishabhJain2018** [comment]
@vkartik97 Can you please resolve the conflicts here?

**Comment by vkartik97** [comment]
@RishabhJain2018 Please review !

**Pull request merged**